tests/functional/adapter/seed_column_types/fixtures.py

```diff
@@ -0,0 +1,111 @@
+_MACRO_TEST_IS_TYPE_SQL = """
+{% macro simple_type_check_column(column, check) %}
+    {% if check == 'string' %}
+        {{ return(column.is_string()) }}
+    {% elif check == 'float' %}
+        {{ return(column.is_float()) }}
+    {% elif check == 'number' %}
+        {{ return(column.is_number()) }}
+    {% elif check == 'numeric' %}
+        {{ return(column.is_numeric()) }}
+    {% elif check == 'integer' %}
+        {{ return(column.is_integer()) }}
+    {% else %}
+        {% do exceptions.raise_compiler_error('invalid type check value: ' ~ check) %}
+    {% endif %}
+{% endmacro %}
+
+{% macro type_check_column(column, type_checks) %}
+    {% set failures = [] %}
+    {% for type_check in type_checks %}
+        {% if type_check.startswith('not ') %}
+            {% if simple_type_check_column(column, type_check[4:]) %}
+                {% do log('simple_type_check_column got ', True) %}
+                {% do failures.append(type_check) %}
+            {% endif %}
+        {% else %}
+            {% if not simple_type_check_column(column, type_check) %}
```

Comment:
We log in the first condition, but not the second. Is that intended? If the behavior should be the same, then I would consider updating `simple_type_check_column()` to handle the negative conditions instead of handling it here.

Reply:
yeah looks like we don't log in second condition because that is the failure mode and we would throw the exception from the `simple_type_check_column` instead of a log showing it passed